### PR TITLE
Add Equatable example

### PR DESCRIPTION
**Pull request by Abhishek01039**
Have added Equatable example in .family (https://riverpod.dev/docs/concepts/modifiers/family#passing-multiple-parameters-to-a-family).

Thanks

### CODE REVIEW

File: website/docs/concepts/modifiers/family.mdx [hunk 1]
```diff
@@ -103,6 +103,17 @@ This includes:
 
 Here's an example using [Freezed]:
 
+<Tabs
+  groupId="family"
+  defaultValue="Freezed"
```
Comment (rrousselGit) [inline website/docs/concepts/modifiers/family.mdx:108]:
```suggestion
  defaultValue="freezed"
```

File: website/docs/concepts/modifiers/family.mdx [hunk 2]
```diff
@@ -130,6 +177,7 @@ Widget build(BuildContext context, ScopedReader watch) {
   ...
 }
 ```
+</TabItem>
```
Comment (rrousselGit) [inline website/docs/concepts/modifiers/family.mdx:180]:
```suggestion
</TabItem>
</Tabs>
```

**Comment by rrousselGit** [comment]
The CI currently does not pass (the doc website doesn't build). Could you fix that? The suggestions I made should fix it I think

File: website/docs/concepts/modifiers/family.mdx (hunk 1 above)
Comment (Abhishek01039) [reply website/docs/concepts/modifiers/family.mdx]: Done

File: website/docs/concepts/modifiers/family.mdx (hunk 2 above)
Comment (Abhishek01039) [reply website/docs/concepts/modifiers/family.mdx:180]: Done

**Comment by Abhishek01039** [comment]
Seems like all test is passed !! @rrousselGit

File: packages/flutter_riverpod/lib/src/builders.dart [hunk 3]
```diff
@@ -232,7 +232,40 @@ class ChangeNotifierProviderBuilder {
...
+  ///
+  ///   final something = watch(
+  ///     exampleProvider(MyParameter(userId: userId, locale: locale)),
+  ///   );
+  ///
+  ///   ...
+  /// }
+  /// ```
+  ///
+  /// ```dart
```
Comment (rrousselGit) [inline packages/flutter_riverpod/lib/src/builders.dart:250]: We need to explain what this code block is about

File: website/docs/concepts/modifiers/family.mdx [hunk 4]
```diff
@@ -101,7 +101,18 @@ This includes:
 - Objects generated with [Freezed] or [built_value](https://pub.dev/packages/built_value)
 - Objects using [equatable](https://pub.dev/packages/equatable)
 
-Here's an example using [Freezed]:
+Here's an example using [Freezed] and [equatable]:
```
Comment (rrousselGit) [inline website/docs/concepts/modifiers/family.mdx:104]: the [equatable] link is broken

Comment (rrousselGit) [reply website/docs/concepts/modifiers/family.mdx:104]: "Here's an example using [freezed]" should be inside the Freezed tab, not above all tabs

Comment (Abhishek01039) [reply website/docs/concepts/modifiers/family.mdx:104]: Added the link equatable

File: packages/flutter_riverpod/lib/src/builders.dart (hunk 3 above)
Comment (Abhishek01039) [reply packages/flutter_riverpod/lib/src/builders.dart:250]: Take `userId` and `locale` externally and give them to the provider.

should be like this ?

**Comment by Abhishek01039** [comment]
Should we need `autoDispose`? in every code snippet?
because you have added `autoDispose` in one of the code snippets.
```
final exampleProvider = Provider.autoDispose.family<Something, MyParameter>((ref, myParameter) {
  print(myParameter.userId);
  print(myParameter.locale);
  // Do something with userId/locale
});
```

**Comment by Abhishek01039** [comment]
image
This one @rrousselGit

**Comment by rrousselGit** [comment]
I've updated the files

Thanks!

**Pull request merged**